src/ObjectStore/v1/Models/Object.php

```diff
@@ -96,11 +96,14 @@ public function retrieve()
      * distinct from fetching its metadata (a `HEAD` request). The body of an object is not fetched by default to
      * improve performance when handling large objects.
      *
+     * @param array $options {@see \OpenStack\ObjectStore\v1\Api::getObject} 
+     *
      * @return StreamInterface
      */
-    public function download(): StreamInterface
+    public function download($options=array()): StreamInterface
```

Comment:
Should be `array  $options = []`. I think @jamiehannaford 's points is that we use square bracket for array.